src/components/ActiveWalletSection/index.tsx

```diff
@@ -120,7 +132,7 @@ export default function ActiveWalletSection() {
       predicate: (query) =>
         query.queryKey[0] === 'activeWallet' ||
         query.queryKey[0] === 'gameplayWallets' ||
-        query.queryKey[0] === getGetExternalEligibilityQueryKey(null)[0] // we don't really care for the questId here
+        Object.values(eligibilityQueries).includes(query.queryKey[0] as string)
```

Comment:
if we set an `'allEligibility` prefix for all the eligibility query keys, we could just invalidate that prefix instead of each individual eligibility query key

so `getGetExternalEligibilityQueryKey` would instead return

```
[allEligibility, externalEligibilityQueryKeyPrefix, questId]
```

and so on for the other eligibility queries